VOTE-2587 Config, templates and po file changes related to string migration

rayestrada opened the pull request:
<!-- Delete any detail that does not apply to this PR! -->

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2587

## Description

This PR has 3 objectives:
1. Replace current hardcoded strings in templates with the dynamic variables
2. Add a new 'Submit button label' field for hardcoded string for search component.
3. Remove translations from the po files.

## Deployment and testing

### Post-deploy steps

1. Pull the latest db from prod to local
2. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/media/add/image and make sure the caption field only accepts simple formatting options
2. visit https://vote-gov.lndo.site/admin/content/block/2 and add a Submit Button Label
3. Test the display of each of these components for migrated content
- Email sign up block
- government banner block
- search component block
- contact identifier block
- partnership block
- registration tool paragraph

## Checklist for the Developer

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask for help! -->
- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [ ] Code is readable and includes appropriate commenting.
- [ ] Code standards and best practices are followed.
- [ ] QA/Test steps were successfully completed, if applicable.
- [ ] Applicable logs are free of errors.

SamiraMSadat commented on web/themes/custom/votegov/templates/block/block-content--partnership.html.twig:
```diff
@@ -0,0 +1,72 @@
+      </div>
+      {# EAC logo #}
+      <div class="vote-eac-partner__logo-img">
+        <a extlinkjs-ignore
```
There needs to a style for the class that limits the size of the image.

SamiraMSadat commented on web/themes/custom/votegov/templates/block/block-content--partnership.html.twig:
```diff
@@ -0,0 +1,72 @@
+    <div class="vote-usagov-partner__contact">
+      <div class="vote-usagov-partner__logo">
+        {# USA.gov link #}
+        <a href="{{ content.field_usagov_logo_link | field_value }}" extlinkjs-ignore>
```
There needs to be a style for the class that limits the size of the image.

rayestrada commented:
I've added new size styles for the imgs. I've also removed the alt text for the U.S.Flag icon and made that decorative following recent updates to the USWDS banner https://designsystem.digital.gov/components/banner/.
